src/main/java/org/folio/bulkops/processor/InstanceUpdateProcessor.java

```diff
@@ -117,8 +166,15 @@ private String buildErrorMessage(boolean recordsUpdated, boolean newValue) {
       MSG_NO_CHANGE_REQUIRED;
   }
 
+  private List<HoldingsRecord> getHoldingsSourceFolioByInstanceId(String instanceId) {
+    return holdingsClient.getByQuery(format(GET_HOLDINGS_BY_INSTANCE_ID_QUERY, instanceId), Integer.MAX_VALUE)
+      .getHoldingsRecords().stream()
+      .filter(holdingsRecord -> !"MARC".equals(holdingsReferenceService.getSourceById(holdingsRecord.getSourceId()).getName()))
```

Comment:
M.b. better use BooleanUtils.isFalse or separate in private method .. Using reverted Boolean logic not easy to read